plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestOnlyNulls.java

```diff
@@ -56,13 +56,13 @@ public void testOnlyNulls()
         HiveColumnHandle column = createBaseColumn(columnName, 0, HiveType.toHiveType(columnType), columnType, REGULAR, Optional.empty());
 
         // match not null
-        try (ConnectorPageSource pageSource = createPageSource(SESSION, parquetFile, ImmutableList.of(column), TupleDomain.withColumnDomains(Map.of(column, notNull(columnType))))) {
```

Comment:
This test would still pass with the old `io.trino.plugin.hive.parquet.ParquetUtil#createPageSource(io.trino.spi.connector.ConnectorSession, java.io.File, java.util.List<io.trino.plugin.hive.HiveColumnHandle>, io.trino.spi.predicate.TupleDomain<io.trino.plugin.hive.HiveColumnHandle>)` method.